.jenkins/common_job_properties.groovy

```diff
@@ -171,6 +180,20 @@ class common_job_properties {
     setPullRequestBuildTrigger(context, commitStatusName, successComment)
   }
 
+  // Enable triggering postcommit runs against pull requests. Users can comment the trigger phrase
+  // specified in the postcommit job and have the job run against their PR to run
+  // tests not in the presubmit suite for additional confidence.
+  static def enablePhraseTriggeringFromPullRequest(def context,
```

Comment:
Lots of boilerplate when there's only single post-commit (nightly) that doesn't want to have this functionality.

I suggest simplifying: make this the default, required behavior, and nightly build can override back manually inside itself.